tests/chainer_tests/training_tests/updaters_tests/test_multiprocess_parallel_updater.py

```diff
@@ -150,18 +150,18 @@ def setUp(self):
     @unittest.skipUnless(mpu.MultiprocessParallelUpdater.available(),
                          'MultiprocessParallelUpdater is not available.')
     def test_update_uses_raw_array(self):
-        ret, stdoutdata, stderrdata = _run_test_snippet('raw_array.py')
+        ret, stdoutdata, stderrdata = _run_test_snippet(
+            'raw_array.py', '@cupy:0')
         assert ret == 0, (
             '[stdout]:{!r}\n'
             '[stderr]:{!r}'.format(stdoutdata, stderrdata))
 
 
 class TestChildReporter(unittest.TestCase):
 
-    def check_with_gpus(self, n_devices):
-        device_ids_str = ','.join([str(n) for n in range(n_devices)])
+    def check_with_devices(self, *devices):
```

Comment:
Why did you change the interface of this method?
The previous one looks better for me.
Basically `*` is a syntactic sugar and does not seem appropriate in this situation.

Reply:
Fixed.